### DISCUSSION TITLE
Gran Insulae holds more residents but requires more

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#827] [Ideas]
I modified my version of Augustus by setting population in a gran insulae to 88 but requires academy access. My guess is that to justify a second type of food the game should provide me a little benefit -- a few more people, do you agree?
Both changes are easily done just my modifying the file c3_model.txt, really trivial:

**Comment by crudelios** [comment]
Since this can be changed using `c3_model.txt`, there's no need to request a code change.

Do note, however, that eventually are going to integrate `c3_model.txt` into the code itself, as managing both original buildings on that model and new buildings on the code is getting really messy.

**Comment by githubfan1** [comment]
These are the changes to c3_model.txt:

![G-Insulae-changes](https://user-images.githubusercontent.com/6599012/221361947-dad1eb30-9c63-4005-9234-e63f5c1a2301.jpg)
